docs/migration/test-cloud/index.md

```diff
@@ -1,13 +1,94 @@
 ---
-title: "Test Cloud Migration"
-author: joemellin
-ms.author: joemel
-ms.date: 04/10/2017
+title: Test Cloud Migration Guide
+description: "Xamarin Test Cloud to AppCenter test tranition guide. Relevant to Xamarin Test Cloud subscribers that will be automatically upgraded from Xamarin Test Cloud to Visual Studio App Center on March 31, 2018"
+keywords: testcloud
+author: glwest
+ms.author: glwest
+ms.date: 2/21/2018
 ms.topic: article
-ms.assetid: 8495359C-F8E6-45B4-8528-C1EFC0A6B247
+ms.assetid: 4a110ae0-21d6-46c4-bcca-963ac0e2c3d7
 ms.service: vs-appcenter
+ms.custom: test
 ---
 
-# Test Cloud Migration
+# Upgrading all Xamarin Test Cloud subscribers to Visual Studio App Center
+
+Microsoft will seamlessly upgrade active paid Xamarin Test Cloud subscriptions to [Visual Studio App Center](http://appcenter.ms). Visual Studio App Center offers the same devices, performance, and features as Xamarin Test Cloud, plus an upgraded API, unified CLI, and new features like saved device sets. It also integrates our automated test lab with our other App Center services, like cloud-based builds, app distribution, crash reporting, user analytics, and push notifications.
+
+## Timeline
+* March 31st, 2018 
+	1. Teams / Orgs / Apps / previous test reports will import into App Center. 
+	2. All your test results that are being uploaded via our XTC CLIs will automatically route your test results into your App Center account.
+	3. The [Xamarin Test Cloud](http://testcloud.xamarin.com) website will redirect to [Visual Studio App Center](http://appcenter.ms).
+* April 30th, 2018
+	1. XTC CLI will be deprecated, everyone must use the [App Center CLI](https://github.com/Microsoft/appcenter-cli) to continue uploading. 
+
+
+## Next steps
+
+Users must log into app center with the same email they used in XTC to benefit from the above import. If a user doesn't have an App Center account, one will be created and user will need to set password. 	
+
+* **If you do not have an App Center account**
+	* [Create](http://appcenter.ms/) one today, use same email as you use in XTC.
+	*  If you do not wish to try App Center before March 31st, you can simply wait until that day and we'll create the account and import your data for you.
+
+* **If you already have an App Center account (email matches XTC)**
+	* Relevant Teams / Orgs / Apps / previous test reports will import into App Center automatically.
+
+
+
+## FAQ
+
+**Q:** *How do I install the App Center CLI?*
+ 
+**A:** We are committed to maintain the CLI until the end of April 2018. As part of the upgrade we will route the results that are generated by your existing Xamarin Test Cloud upload command to App Center through April 30th, 2018. 
+
+In order to start uploading to App Center, you must install the App Center CLI. To install it, please use:
```

Comment:
"To install it, open a terminal window and execute the following command:" is a more specific instruction.